resource/api/swagger.yml

```diff
@@ -225,24 +225,26 @@ paths:
   /websiteCreator/prepare:
     put:
       operationId: websiteCreatorPrepare
+      consumes:
+        - multipart/form-data
       parameters:
-        - in: body
-          name: body
+        - in: formData
+          name: url
+          type: string
           required: true
-          schema:
-            type: object
-            required:
-              - url
-              - nickname
-            properties:
-              url:
-                description: URL without '.', capitals letters and specifics characters
-                type: string
-                x-nullable: false
-              nickname:
-                description: Wallet's nickname to be used for preparing the website
-                type: string
-                x-nullable: false
+
+          description: URL without '.', capitals letters and specifics characters
```

Comment:
can you add a pattern field to filter forbidden characters ? And test how the API react when you use one ?